VAMobile/src/components/FormWrapper/FormFields/VASelector.test.tsx

```diff
@@ -2,12 +2,11 @@ import 'react-native'
 import React from 'react'
 // Note: test renderer must be required after react-native.
 import 'jest-styled-components'
-import { act, ReactTestInstance } from 'react-test-renderer'
+import { ReactTestInstance } from 'react-test-renderer'
```

Comment:
line 5 should be removed and tests updated accordingly

Reply:
Updated.